IonDotnet/Tree/IIonValue.cs

```diff
@@ -1,11 +1,13 @@
 ﻿using System.Collections.Generic;
-using IonDotnet.Tree.Impl;
 
 namespace IonDotnet.Tree
 {
-    public interface IIonValue
+    public interface IIonValue : IIonNull, IIonBool, IIonInt, IIonFloat,
```

Comment:
Should we put the inherited interfaces in alphabetical order?

Reply:
I think it's fine as is, since it's in the BinaryTypeId of the IonTypes